packages/react-reconciler/src/__tests__/ReactTransition-test.js

```diff
@@ -168,6 +168,38 @@ describe('ReactTransition', () => {
     }
   }
 
+  it('commits outside a suspended Suspense boundary', async () => {
+    const neverResolve = new Promise(() => {});
+
+    function App({step, shouldSuspend}) {
+      return (
+        <>
+          <Text text={`A${step}`} />
+          <Suspense fallback={<Text text={`Loading${step}`} />}>
+            {shouldSuspend ? <AsyncText text={`B${step}`} /> : null}
+          </Suspense>
+        </>
+      );
+    }
+
+    const root = ReactNoop.createRoot();
+
+    await act(() => {
+      root.render(<App step={0} shouldSuspend={false} />);
+    });
+
+    assertLog(['A0']);
+    expect(root).toMatchRenderedOutput('A0');
+
+    await act(() => {
+      startTransition(() => {
+        root.render(<App step={1} shouldSuspend={true} />);
+      });
+    });
+    assertLog(['A1', 'Suspend! [B1]', 'Loading1']);
+    expect(root).toMatchRenderedOutput('A1');
```

Comment:
We're currently not committing a partial tree so this ends up as `A0`.

React DevTools relies on the fact that `A1` is committed though so it breaks. Specifically, the Promise we suspend on will only resolve once an Effect outside the Suspense boundary ran. But since we don't commit, we also never flush that particular passive effect.